src/Products/Common/Resources/Resources.cs

```diff
@@ -63,17 +62,17 @@ public ExceptionEntity GenerateException(System.Exception ex)
         /// <param name="ex">Exception</param>
         /// <param name="password">string</param>
         /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex, String password)
+        public ExceptionEntity GenerateException(Exception ex, String password)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=5153088)